test/libsolidity/smtCheckerTests/natspec/safe_assert_false_positive_pure.sol

```diff
@@ -0,0 +1,29 @@
+contract C {
```

Comment:
@hrkrshnn added this test, I think that's what you meant.
I left `g` as mutable just to really check that the other functions do not change the state